README.md

```diff
@@ -77,13 +77,13 @@ With each string, we want to execute a network request. Luckily, RAC offers an
 
 ```swift
 let searchResults = searchStrings
-    .flatMap(.Latest) { query in
+    .flatMap(.Latest) { (query: String) -> SignalProducer<(NSData, NSURLResponse), NSError> in
```

Comment:
I think this is useful in documentation in any case: type inference is good, but better to be explicit to help people understand sample code!